### DISCUSSION TITLE
Attach Media Group to Message

### DISCUSSION THREAD

**Discussion by AlexanderFalkenberg** [def-studio/telegraph#663] [Q&A]
First of all, thank you for implementing the mediaGroup method! @MarioGattolla. 

Is it possible to attach a Media Group to a message? Are there any plans to implement it?

**Comment by MarioGattolla** [comment]
Hi @AlexanderFalkenberg  , unfortunately sendMediaGroup method doesn't support the "caption" attribute (https://core.telegram.org/bots/api#sendmediagroup)  like other attachments. You can try with the "reply_parameters" attribute, this will link the MediaGroup to a message  like this : 

`    $last_message_id = $telegraphChat->message('message')->send()->telegraphMessageId();
`


    $telegraphChat->mediaGroup([
        [
            'type' => 'photo',
            'media' => 'https://test.jpg',
        ],
        [
            'type' => 'photo',
            'media' => 'https://test.jpg',
        ],
    ])->withData('reply_parameters', ['message_id' => $last_message_id])->send();

**Comment by devope** [comment]
Actually it is possible, just add `caption` key to the first only attachment.
